ha/util/conf_store.py

```diff
@@ -221,3 +221,30 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_pods(index):
+        """
+        Returns machine_id for control-node pod.
+        Args:
+            index(str): index of conf file
+            parent key : node
+            search key: type
+            search val: control_node
+        Returns:
+            list: list of node_ids
+
+        >>> Conf.search("cortx", 'node', 'type', 'control_node')
+        ['node>a3710eee36644f2fb60ba19486664495>type']
+        """
+        node_ids = []
+        try:
+            node_key = Conf.search(index, GconfKeys.NODE_CONST.value,
+                                   GconfKeys.TYPE_CONST.value, GconfKeys.CONTROL_NODE.value)
+            for key in node_key:
+                value = key.split('>')[1]
+                node_ids.append(value)
+            Log.info(f"Fetched control node machine-id: {node_ids}")
+        except Exception as e:
+            Log.error(f"Unable to fetch control node machine-id. Error:{e}")
+        return node_ids
```

Comment:
Only one control node is expected in the system. So list is not needed.(in fact we should not get more than 1 ids)
@gargpalak048  please get this confirmed from @shaileshsvc and update the PR